docs/source/conventions.rst

```diff
@@ -14,6 +14,8 @@ We adopt the following system for branching and merging:
 6. Commit often.
 7. Branch often. Merge after completion of the task for this branch, then delete the branch.
 8. Branch only from master or from develop.
+9. HOTFIX workflow: branch from master into hotfixNNN_title (NNN = ticket number), update tests,
+   fix the bug, increment micro version, test, merge into master and into develop, delete branch.
```

Comment:
No, thought this was what we agreed not to do.
Thought this was the procedure: Merge into master, then merge master into develop.